map2loop/thickness_calculator.py

```diff
@@ -244,7 +243,7 @@ def compute(
         # get the sampled contacts
         contacts = geopandas.GeoDataFrame(map_data.sampled_contacts)
         # build points from x and y coordinates
-        geometry2 = contacts.apply(lambda row: Point(row.X, row.Y), axis=1)
+        geometry2 = contacts.apply(lambda row: shapely.Point(row.X, row.Y), axis=1)
```

Comment:
It is significantly more efficient to use `gpd.points_from_xy(df.x, df.y)` to create a geometry dataframe

Reply:
In https://github.com/Loop3D/map2loop/pull/91/commits/2f931c59025997133aeea9c10f725858613b7f6b